package.json

```diff
@@ -5,8 +5,8 @@
   "main": "client/index.js",
   "dependencies": {
     "body-parser": "^1.19.0",
-    "canvas2svg": "^1.0.16",
     "cookie-parser": "^1.4.4",
+    "d3": "^5.16.0",
```

Comment:
Previously, we were reaching out to the internet to download these resources at runtime.  Instead, we can just pull them in via `npm` and bundle them directly into `bundle.js` via `browserify` (see also: c6bd1b66071f1129b3fb7bacc5098ce757cb6e51).